templates/master/00-master/_base/files/usr-local-bin-openshift-recovery-tools-sh.yaml

```diff
@@ -148,6 +159,16 @@ contents:
       fi
     }
 
+    remove_kube_static_resources() {
+      COUNT=$(ls -d "${CONFIG_FILE_DIR}"/static-pod-resources/kube-apiserver-pod-[0-9]* 2> /dev/null | wc -l) || true
+      if [ "$COUNT" -gt 1 ]; then
+         echo "Removing etcd certs.."
+         rm -rf "${CONFIG_FILE_DIR}"/static-pod-resources/kube-apiserver-pod-[0-9]*
```

Comment:
why do we need to delete all the revision directories? I know we only need the revision that was backed up, but to be extra safe would be a good idea to only delete the ones after the revision number that was backed up?

Reply:
I am working on deleting only later revisions. Look for another push.